pkg/validation/validation.go

```diff
@@ -107,6 +107,15 @@ func validateLogBatchLimits(structLevel validator.StructLevel) {
 	}
 }
 
+// SetTag must have either a run_id or a run_uuid present.
+func validateSetTagRunIDExists(structLevel validator.StructLevel) {
```

Comment:
This looks good, but does that produce the correct error message?

Reply:
I believe the error message isn't getting checked anywhere. Should we perhaps extend the Python tests to make sure this is in order? Or is this something we handle in the Go tests?

I did notice that unlike the other tests (for validation), the run_uuid/run_id test doesn't actually care about an error message. I'm guessing that the only undesirable case is when we have neither of them (and we would just throw an empty `run_id` error). However, there doesn't seem to be a test for this.